Modules/_testcapi/dict.c

```diff
@@ -140,13 +221,19 @@ dict_getitemwitherror(PyObject *self, PyObject *args)
 }
 
 
+/*[clinic input]
+_testcapi.dict_getitemref
+    obj: object
+    attr_name: object
+    value: object
```

Comment:
This is not correct; `dict_getitemref` only takes two positional-only arguments: `obj` and `attr_name`.